api/src/models/draft-create.ts

```diff
@@ -0,0 +1,15 @@
+import { getLogger } from '../utils/logger';
+
+const defaultLog = getLogger('models/draft-create');
+
+export class PostPutDraftObject {
```

Comment:
Is there a reason it's called `PostPutDraftObject` and not `PutDraftObject`?

Reply:
The naming feels odd ... but this constructor is used for both the create and update endpoints.